index.html

```diff
@@ -1,12 +1,212 @@
+            </aside>
+        </div>
+    </section>
+    <section class="footer">
+        <section class="flexContent">
+            <div class="project">
+                <aside class="aside aside-1">
+                    <h1 class="h1">
+                        About Myself
+                    </h1>
+                    <p>
+                        I'm a software developer! I can help you build a product, feature or website look through some of my
+                        work and experience! If you like what you see and have a project you need coded, don't hestiate to
+                        contact me.
+                    </p>
+                    <h5 class="paragraph">LET'S CONNECT</h5>
+                    <div class="link-connect">
+                        <ul>
+                            <li>
+                                <a href="https://www.linkedin.com/in/menendezon">
+                                    <img src="images/linkedin.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="https://twitter.com/menendezon">
+                                    <img src="images/twitter.png" alt="" srcset="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="mailto://menendezon@gmail.com">
+                                    <img src="images/gmail.png" alt="" srcset="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="https://github.com/MenendezON">
+                                    <img src="images/github.png" alt="" srcset="">
+                                </a>
+                            </li>
+                        </ul>
+                    </div>
+                    <button class="btn">Get my resume</button>
+                </aside>
+                <aside class="aside aside-2">
+                    <div class="qualification">
+                        <div class="skl">Languages</div>
+                        <ul>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                        <div class="skl">Frameworks</div>
+                        <ul>
+                            <li>Laravel</li>
+                            <li>Bootstrap</li>
+                            <li>Spring</li>
+                        </ul>
+                        <div class="skl">Skills</div>
+                        <ul>
+                            <li>Project management</li>
+                            <li>IT Network</li>
+                            <li>Problem-solving</li>
+                        </ul>
+                    </div>
+                </aside>
+            </div>
+        </section>
+    </section>
```

Comment:
- [ ] Good job with the additional requirements but really, this is for future milestones :+1: